vim.md

```diff
@@ -0,0 +1,196 @@
+# Vim and Markdown
+
+## Vim
+
+Vim is described as a 'highly configurable text editor for efficiently creating
+and changing any kind of text'
+
+Vim is included under the alias vi on most UNIX systems, as well as OS-X
+
+Vim is designed with the intent to have functions easily accessable and to never
+require the user to remove their hands from the home row.
+
+### Getting started with Vim
+
+To start vim without opening a specific file simply type `vim` on the terminal.
+To open a file, write `vim filename`, in the case of this file, one would type
+`vim vim.md`, like you likely already have.
+
+to quit vim, simply hit `esc` a couple of times, then type `:q` and hit enter.
+Hitting escape ensures that you are not in insert mode. if this gives you an
+error, typing `:q!` will force exit, but will discard all changes.
+
+to save a file in vim, go to normal mode and type `:w` this will save the file
+directly back to the same filename. this can be combined with the `q` command
+with `:wq`, saving and quitting at the same time.
+
+#### Modes of Vim
+
+Before we jump into movement, we should quickly go over the 2 modes of vim; 
+insert and visual. You can enter these modes at any time while not in another
+mode, and can get out of them by hitting the `esc` key.
+
+##### Insert
+
+While in insert mode, vim acts as a basic text editor, type on the keyboard and
+text appears in the file. Cursor movement can be achived with the arrow keys.
+insert mode can be accessed with the `i` key, and will start the insertion at
+the exact character that the cursor is on. if you wish to start at the character
+after, use the `a` key instead
+- `i` enter insert mode at current character
+- `a` enter insert mode after current character
+
+##### Visual
+
+Visual mode is akin to selecting text in a graphical editor, though with some
+more features. For now, Enter visual mode with `v` selecting the current
+character, and move the cursor to select more text.
+- `v` enter visual mode on a character-by-character basis
+- `V` enter visual mode on a line-by-line basis
+
+This mode acts like the copy-paste mode of vim.
+
```

Comment:
Maybe adding a section of normal mode, just to specify it's the mode you're in after hitting the `esc' key so that it's been defined when it's referenced in a few paragraphs.